config/envs/unit_test.py

```diff
@@ -19,19 +19,12 @@ class UnitTestConfig(Config):
     AUTHENTICATOR_HOST = "http://localhost:5000"
 
     # Azure Active Directory Config
-    # This secret is only used for local testing purposes
+    AZURE_AD_CLIENT_ID = "d8be82a8-541c-4768-9296-84bd779a24d9"
     AZURE_AD_CLIENT_SECRET = "nmq8Q~acUEOPWmjfvbOEQLPZy2M38yLe1PEh_cS2"
```

Comment:
I really don't think this should be public, even for a test instance, so I think it would be better to rotate those credentials, mock the requests as you suggest, or pass them through as secrets using something like dotenv,